app/scripts/filters/resources.js

```diff
@@ -1029,32 +1029,66 @@ angular.module('openshiftConsole')
         return 'Terminating';
       }
 
-      var reason = pod.status.reason || pod.status.phase;
+      var initializing = false;
+      var reason;
 
-      // Print detailed container reasons if available. Only the last will be
-      // displayed if multiple containers have this detail.
-      angular.forEach(pod.status.containerStatuses, function(containerStatus) {
-        var containerReason = _.get(containerStatus, 'state.waiting.reason') || _.get(containerStatus, 'state.terminated.reason'),
-            signal,
-            exitCode;
+      _.forEach(pod.status.initContainerStatuses, function(initContainerStatus) {
+        var initContainerState = _.get(initContainerStatus, 'state');
 
-        if (containerReason) {
-          reason = containerReason;
-          return;
+        if (initContainerState.terminated && initContainerState.terminated.exitCode === 0) {
+          // initialization is complete
+          return true;
```

Comment:
`return true` will short-circuit here and skip the remaining init containers. Since this container was successful, it's not what we want. We want to short0circuit only when we find one that is not complete or failed.

I think you want just `return;`